index.html

```diff
@@ -8,7 +8,32 @@
     <link href="vendor/font-awesome-4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="dist/css/jsflap.css">
     <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=no">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
Please do not include jQuery or the below library, we are already using angular which includes a lite version of jQuery already, make an angular directive if you really need the access.